test/ex_bitmex/rest/orders/cancel_bulk_test.exs

```diff
@@ -0,0 +1,38 @@
+defmodule ExBitmex.Rest.Orders.CancelBulkTest do
+  use ExUnit.Case, async: true
```

Comment:
This should be `async: false` to avoid hitting nonce errors when recording multiple cassette's